web/src/views/SubmissionPortal/store/index.ts

```diff
@@ -149,9 +149,11 @@ const templateChoiceDisabled = computed(() => {
 
 const tabsValidated = ref({} as Record<string, boolean>);
 watch(templateList, () => {
+  const newTabsValidated = {} as Record<string, boolean>;
   forEach(templateList.value, (templateKey) => {
-    tabsValidated.value[templateKey] = false;
+    newTabsValidated[templateKey] = false;
   });
+  tabsValidated.value = newTabsValidated;
 });
```

Comment:
@pkalita-lbl if you have a better idea of how to construct this object in general, let me know. This solves the problem of abandoning keys in `tabsValidated` that are no longer present in `templateList`, but I'm open to other ways to solve this.

Reply:
No I think this makes sense to me